### PR TITLE
Add template docs structure

### PR DESCRIPTION
**Pull request by m-green**
## What’s changed
Replaced the default 'Hello world' page with a template docs structure based on the new version of our [Documenting APIs](https://www.gov.uk/guidance/how-to-document-apis) guidance.

## Why
The template will help API teams get started on documentation with a structure that meets our documentation standard and GOV.UK content and style guidelines. The data community have recently discussed the need for a template docs structure.

### CODE REVIEW

**Review by laurahiles — APPROVED** [review]

File: optional/config/tech-docs.yml.tt [hunk 1]
```diff
@@ -21,15 +21,15 @@ enable_search: false
 ga_tracking_id:
 
 # Enable multipage navigation in the sidebar
-multipage_nav: false
+multipage_nav: true
 
 # Enable collapsible navigation in the sidebar
 collapsible_nav: false
```
Comment (lfdebrux) [inline optional/config/tech-docs.yml.tt:27]: Is there any reason not to use collapsible navigation?

Right now it looks a bit weird (in my opinion), with the headings for the support links trailing at the end:

Screenshot 2022-03-29 at 13 39 25

File: optional/source/how-to-do-a-task-1.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: How to [name of task]
```
Comment (lfdebrux) [inline optional/source/how-to-do-a-task-1.html.md.erb:2]: Keeping three duplicates of this page template seems like additional maintenance burden we could avoid; could we instead just tell users that they can copy this template as needed?

**Review by lfdebrux — COMMENTED** [review]
Works for me!

Just two comments, but they aren't major issues.

File: optional/config/tech-docs.yml.tt (hunk 1 above)
Comment (m-green) [reply optional/config/tech-docs.yml.tt:27]: Ah yes good call! I'll fix that.

File: optional/source/how-to-do-a-task-1.html.md.erb (hunk 2 above)
Comment (m-green) [reply optional/source/how-to-do-a-task-1.html.md.erb]: Hi @lfdebrux - good suggestion, thank you! I've updated that now. Could you review and approve if you're happy?

**Review by lfdebrux — APPROVED** [review]

**Pull request merged**